src/Emulator/Main/Backends/Display/IPixelBlender.cs

```diff
@@ -11,12 +12,19 @@ namespace Antmicro.Renode.Backends.Display
 {
     public interface IPixelBlender
     {
-        void Blend(byte[] backBuffer, byte[] frontBuffer, ref byte[] output, Pixel background = null, byte backBufferAlphaMultiplayer = 0xFF, byte frontBufferAlphaMultiplayer = 0xFF);
-        void Blend(byte[] backBuffer, byte[] backClutBuffer, byte[] frontBuffer, byte[] frontClutBuffer, ref byte[] output, Pixel background = null, byte backBufferAlphaMultiplayer = 0xFF, byte frontBufferAlphaMultiplayer = 0xFF);
+        void Blend(byte[] backBuffer, byte[] frontBuffer, ref byte[] output, Pixel background = null, byte backBufferAlphaMultiplayer = 0xFF, PixelBlendingMode backgroundBlendingMode = PixelBlendingMode.MULTIPLY, byte frontBufferAlphaMultiplayer = 0xFF, PixelBlendingMode foregroundBlendingMode = PixelBlendingMode.MULTIPLY);
+        void Blend(byte[] backBuffer, byte[] backClutBuffer, byte[] frontBuffer, byte[] frontClutBuffer, ref byte[] output, Pixel background = null, byte backBufferAlphaMultiplayer = 0xFF, PixelBlendingMode bgBlendingMode = default(PixelBlendingMode), byte frontBufferAlphaMultiplayer = 0xFF, PixelBlendingMode foregroundBlendingMode = PixelBlendingMode.MULTIPLY);
```

Comment:
`PixelBlendingMode bgBlendingMode = default(PixelBlendingMode),` -> I'd go with the version used in other paremeters:
`PixelBlendingMode backgroundBlendingMode = PixelBlendingMode.MULTIPLY`